### DISCUSSION TITLE
MonitorControl v4.3.3

### DISCUSSION THREAD

**Discussion by waydabber** [MonitorControl/MonitorControl#1647] [General]
This update contains some minor fixes and changes to the previous app version which introduced macOS Sequoia compatibility and included some other changes.

For more features please consider switching to **[BetterDisplay](https://betterdisplay.pro)**. 

Please support this project at our **[opencollective site](https://opencollective.com/monitorcontrol/donate)** for continued development.

Due to a change in signatures, if you are still running v4.2.0, auto-update will not work. You need to download and install this version manually.

## What's Changed

- Fixed: "Dead pixel(s)" at the left bottom of the external screen appears when app is running - @waydabber
- Fixed: App menu in the menu bar is showing icons when text is selected — and vice versa - @waydabber
- Fixed: Menu Bar Panel overflowing in Spanish - @waydabber
- Renamed all occurenses of `Preferences` to `Settings` + update to the latest version of [sindresorhus/Settings ](https://github.com/sindresorhus/Settings)- @waydabber

<hr /><em>This discussion was created from the release <a href='https://github.com/MonitorControl/MonitorControl/releases/tag/v4.3.3'>MonitorControl v4.3.3</a>.</em>

**Comment by thesagarmahajan** [comment]
Perfectly orking.

**Comment by bsalemnader** [comment]
Not working. I have an MSI MP242A monitor, and I can only adjust the brightness, not the sound.

**Reply by waydabber** [reply]
Make sure the display is not in HDR mode.

**Reply by AGme99** [reply]
I have two ThinkVision displays T27h-20 and T27h-30 connnected to my mac (m4) via usb-c. one of the displays is connected to an external sound system via 3.5 mm jack and audio cable.
I can adjust the brightness on both but not the volume. any remedy?

**Comment by ccuevasd** [comment]
I am so thankful for this app, so neat and works perfectly!

**Comment by FelixTheKitten** [comment]
I am not sure what to click or press to "apply" advanced settings.  For example, if I set DDC max to 255 and DDC min to 0 nothing happens to the display brightness.  If I hit "Reset settings" it actually applies the changes for a fraction of a second and then immediately reverses them - display gets brighter for less than 1 second.  If I get out of the "Displays" tab and go into the "General" tab, I can get it to apply the changes in DDC levels by toggling either "Combine hardware and software dimming" or by toggling "Allow zero brightness via software or combined dimming".  However, if I use the slider to adjust the display level, these changes in DDC max are eliminated and the maximum level is dim again.  Is there a way to "apply" the changes in DDC levels so they do not get obliterated by using the brightness slider?   I am running MonitorControl 4.3.3 build 7123 on a 2023 MacBook Air (M2 chip) running Sonoma 14.0

**Comment by ghost** [comment]
MacOS 15.5系统，调节亮度默认内建显示器，鼠标放置外接显示器时无法调节，好像获取不到鼠标位置

**Comment by king6731253** [comment]
I am so desire to see any update for this app LoL

**Comment by waydabber** [comment]
Yeah, sorry @king6731253, I started working on a MonitorControl 5 derived from [BetterDisplay](https://betterdisplay.pro) but with a simplified feature set (matching the feature-set of the current MonitorControl app) – but I am just overwhelmed with all the stuff I should do so the progress is a bit stalled. For proper Tahoe support you can use BetterDisplay meanwhile, it has all the features of MonitorControl for free and lots more.

**Comment by SamASakin** [comment]
if not working Try this and REstart. worked for me System Settings → Privacy & Security → Accessibility
enabel the MONITOR CONTROL

**Comment by LifeFlavor** [comment]
Mac OS Tahoe 26.4, seems cannot control extended screen...

**Comment by Elev8matrix** [comment]
Wow, this works great! I didn't like having to use the notch under the monitor all of the time to raise and lower the volume. The brightness options are a plus! One question, when I raise and lower my volume, it adjusts by 6 each time. Is there a way to change this increment? It's not a huge problem, I can use the slider in the menu bar also. Just thought I'd ask.

**Comment by limadiawala** [comment]
I use Samsung's M70F Monitor, I am noit able to control the volume nor the brightness, it only dims via software diming. In my Office I use LG's Monitor, there it works well

Is there anything I need to look out for in the settings or the hardware of the monitor?
